zerver/lib/users.py

```diff
@@ -222,18 +222,31 @@ def access_bot_by_id(user_profile: UserProfile, user_id: int) -> UserProfile:
         raise JsonableError(_("Insufficient permission"))
     return target
 
-def access_user_by_id(user_profile: UserProfile, user_id: int,
-                      allow_deactivated: bool=False, allow_bots: bool=False,
-                      read_only: bool=False) -> UserProfile:
+def access_user_by_id(
+    user_profile: UserProfile,
+    *,
+    target_user_id: int,
```

Comment:
Should this be before the `*`?  It seems like most naturally a required argument.

Reply:
I have put `target_user_id` above, `*` making it a possible to pass it without using keyword notation.